### DISCUSSION TITLE
Why is the 'request' parameter not passed to 'user_manager.delete' in the 'delete' method of the router?

### DISCUSSION THREAD

**Discussion by hgalytoby** [fastapi-users/fastapi-users#1256] [General]
I noticed that `user_manager.delete` can accept a `request` parameter.

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L469

However, when I looked at the delete method in the router, I observed that the request parameter is not being passed when using `user_manager.delete`.

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/router/users.py#L223

I wonder why the `request` parameter is not being passed to `user_manager.delete` in the delete method of the router?

**Comment by frankie567** [comment]
Well, that's a bug 🙃 Good catch!

**Reply by hgalytoby** [reply]
Should I just add the parameter `request: Request` and pass it to `user_manager.delete` to fix this? I haven't merged a project on GitHub before. Would you mind if I give it a try? If it's okay, this will be my first contribution.

**Reply by frankie567** [reply]
Yes, that's exactly it! I suggest you go through this GitHub Tutorial to know how to correctly fork the repository, make your changes and open a PR: https://docs.github.com/en/get-started/quickstart/contributing-to-projects

Will be happy to review your PR :)

**Reply by frankie567** [reply]
Opened the issue #1257 to track this!

**Reply by hgalytoby** [reply]
Got it! I understand. I'll give it a try later as I'm about to leave for home from work.
